src/EventTickets/Actions/RegisterEventsMenuItem.php

```diff
@@ -26,6 +29,18 @@ public function __invoke()
      */
     public function render()
     {
+        if(isset($_GET['id'])) {
```

Comment:
@pauloiankoski alternatively, we could use an additional query parameter such as `?new`, instead of re-using the `?id` query parameter.

Reply:
This would probably be my suggestion, to use a different query parameter for deep-linking the create modal.

Reply:
Either option works for me. Let's go with whatever is your preference then.

Reply:
I think it'll be easier to follow if we use a different query parameter for the deep-link.

Reply:
In that case, we are done here and I'll implement the changes on my PR.